app/namespace.go

```diff
@@ -23,6 +24,7 @@ const (
 	namespaceRegionFlagName          = "region"
 	CaCertificateFlagName            = "ca-certificate"
 	CaCertificateFileFlagName        = "ca-certificate-file"
+	CaPrivateKeyFileFlagName         = "ca-private-key-file"
```

Comment:
[Nit] worth aligning with tctl (except we use hyphens instead of '-')? Specifically do we need the word "private" here?